prometheus_client/__init__.py

```diff
@@ -434,6 +437,33 @@ def do_GET(self):
         self.wfile.write(generate_latest(REGISTRY))
 
 
+def build_pushgateway_url(job, instance=None, host='localhost', port=9091,
+                          use_fqdn_as_instance=True):
```

Comment:
I'd remove all of the instance logic, it's a very rare use case and causes confusion about how the main uses cases are meant to work